src/Dev/FixtureBlueprint.php

```diff
@@ -70,6 +72,17 @@ public function __construct($name, $class = null, $defaults = [])
         $this->defaults = $defaults;
     }
 
+    public function getFactory(): FixtureFactory
```

Comment:
There's a private static $dependencies that includes 'factory' - the get/set with a private is a pretty standard way to handle this in the reset of the codebase